powertools-large-messages/src/main/java/software/amazon/lambda/powertools/largemessages/internal/LargeSNSMessageProcessor.java

```diff
@@ -0,0 +1,39 @@
+package software.amazon.lambda.powertools.largemessages.internal;
+
+import com.amazonaws.services.lambda.runtime.events.SNSEvent.MessageAttribute;
+import com.amazonaws.services.lambda.runtime.events.SNSEvent.SNSRecord;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LargeSNSMessageProcessor extends LargeMessageProcessor<SNSRecord> {
```

Comment:
Doesn't need to be public?

Reply:
initially I wanted to be able to use them out of the annotation context, but I've abandoned, so I guess we can put it package
```suggestion
class LargeSNSMessageProcessor extends LargeMessageProcessor<SNSRecord> {
```